lostfiles.py

```diff
@@ -4,6 +4,8 @@
 import argparse
 import itertools
 import os
+import psutil
+import sys
```

Comment:
This import is unused and should be removed

Reply:
remove import sys